src/viam/sdk/CMakeLists.txt

```diff
@@ -193,7 +193,7 @@ target_link_libraries(viamsdk
   PUBLIC ${VIAMCPPSDK_PROTOBUF_LIBRARIES}
   PRIVATE ${VIAMCPPSDK_GRPCXX_REFLECTION_LIBRARIES}
   PUBLIC Boost::headers
-  PRIVATE Boost::log
+  PUBLIC Boost::log
```

Comment:
Interestingly, external code (like the example module code) fails to compile unless we make the boost log library `PUBLIC` here.